src/k8s/pkg/k8sd/features/cilium/network.go

```diff
@@ -88,9 +88,12 @@ func ApplyNetwork(ctx context.Context, snap snap.Snap, cfg types.Network, _ type
 			},
 		},
 		"nodePort": map[string]any{
-			"enabled": true,
+			"enabled":           true,
+			"enableHealthCheck": false,
```

Comment:
could you add a comment, why we disable this?

Reply:
Maybe we can add a link to the upstream docs:
https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#kube-proxy-hybrid-modes:~:text=nodePort.enableHealthCheck%20to%20false%2C%20so%20that%20the%20Cilium%20agent%20does%20not%20start%20the%20NodePort%20health%20check%20server